components.yaml

```diff
@@ -1,22 +1,36 @@
 components:
   multus:
     url: "https://github.com/intel/multus-cni"
-    commit: "bfaf22964b51b93b08dcb1f10cbd8f8cb9a3603a" # v3.4.1
+    commit: "bfaf22964b51b93b08dcb1f10cbd8f8cb9a3603a"
+    metadata:
```

Comment:
Is the `metadata` level needed ?

Reply:
We currently have this `# v3.4.1` comment. 
I want to keep it up to date, but in order to change it, it needs to have a proper field in the yaml.
@phoracek noted that since it is not really used, but only for human eyes to understand exactly what version is used, he [suggested](https://github.com/kubevirt/cluster-network-addons-operator/pull/479#discussion_r459917991) I change it to make it clearer it's only metadata. 
What would you recommend as the best way to keep this?

Reply:
Is the bumping bot going to change that ? if this is the case this is not metadata.

Reply:
it will change it but only for visibility.  no one actually uses this parameter.

Reply:
Removing complexity is always good I see it just as another tag flating this will help and we are not going to need to filter just by `metadata` fields.

Reply:
following conversation, will change to a flat `metadata`